service/src/main/kotlin/app/cash/backfila/ui/components/NavBar.kt

```diff
@@ -1,52 +0,0 @@
-package app.cash.backfila.ui.components
-
-import app.cash.backfila.ui.pages.ServiceIndexAction
-import app.cash.backfila.ui.pages.ServiceShowAction
-import kotlinx.html.TagConsumer
-import kotlinx.html.a
-import kotlinx.html.div
-import kotlinx.html.img
-import kotlinx.html.nav
-
-fun TagConsumer<*>.NavBar(currentPath: String) {
```

Comment:
We don't need this?

Reply:
Unused from initial work, replaced with the DashboardPageLayout